### PR TITLE
SmartAdServer main script

### PR DESCRIPTION
**Pull request by kendrick-k**
for review !

### CODE REVIEW

File: libraries/smartadserver/smartadserver/v1/Tag.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+//:include tagsdk-current.js
+
+qubit.opentag.LibraryTag.define("smartadserver.smartadserver.v1.Tag", {
+	getDefaultConfig: function () {
```
Comment (peterfronc) [inline libraries/smartadserver/smartadserver/v1/Tag.js]: master version is still using the `config: {` static object

File: libraries/smartadserver/smartadserver/v1/Tag.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+			url: "http://r.sascdn.com/diff/js/smart.js",
+			usesDocWrite: true,
+			parameters: [{
+				name:"SmartAdServer domain",
+				description:"SmartAdServer domain : ex ww691 (ww691.smartadserver.com)",
+				token:"sas_domain",
+				uv:"universal_variable.adserver.sas_domain"
+			}]
+		};
+		/*~DATA*/
```
Comment (peterfronc) [inline libraries/smartadserver/smartadserver/v1/Tag.js]: data marker should wrap the config object internals only (move it after parameters array)

**Comment by kendrick-k** [comment]
hope it's fine

**Comment by peterfronc** [comment]
All looks nice. Ill be soon merging it into our system.

**Comment by kendrick-k** [comment]
:100:

Could you advice for the integration of the Ad itself which uses this library. Calls in the body are done that way : 

<script type="text/javascript">
sas.call("std", {
   siteId: 000000,
   pageId: 111111,
   formatId: 222222,
   target: ""
});
</script>

<noscript>
<a href="http://ww691.smartadserver.com/ac?jump=1&nwid=691&siteid=000000&pageid=111111&fmtid=222222&visit=m&tmstp=[timestamp]&out=nonrich" target="_blank">
    
</a>
</noscript>

Is it possible to integrate it at 100% to OpenTag (I suppose without the noscript), by injecting the result into a div ?

**Comment by peterfronc** [comment]
Ill be merging/testing today all pull requests.

**Comment by kendrick-k** [comment]
see Bundle tag add #62

**Pull request closed without merging**